dotCMS/src/main/java/com/dotcms/rest/api/v1/index/ESIndexResource.java

```diff
@@ -67,392 +67,345 @@
 @Path("/v1/esindex")
 public class ESIndexResource {
 
+    private enum IndexAction{
+        ACTIVATE,
+        DEACTIVATE,
+        CLEAR,
+        OPEN,
+        CLOSE;
+        
+        public static IndexAction fromString(String actionStr) {
+            for(IndexAction action: IndexAction.values()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'action' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)